cron_test.go

```diff
@@ -189,6 +189,35 @@ func TestRepeatLimit(t *testing.T) {
 	assert.Nil(t, cron.GetJob("test-repeat-limit"))
 }
 
+// Job with repeat limit in ISO8601.
+func TestRepeatWithISO8601(t *testing.T) {
+	calledCount := atomic.Int32{}
+
+	cron, err := New(
+		WithNamespace(randomNamespace()),
+		WithTriggerFunc(func(ctx context.Context, m map[string]string, p *anypb.Any) (TriggerResult, error) {
+			calledCount.Add(1)
+			return OK, nil
+		}))
+	if err != nil {
+		t.Fatal("unexpected error")
+	}
+	ctx, cancel := context.WithCancel(context.Background())
+	cron.AddJob(ctx, Job{
```

Comment:
Returned error?

Reply:
Yes, the original code did not return error, new one does. Refactored the tests to assert for no error.